src/services/generate-default-data.ts

```diff
@@ -200,7 +200,11 @@ function parseFormSubmissionModel(
   return parseUnknownAsRecord(value, (record) => {
     return elements.reduce<FormSubmissionModel>((model, element) => {
       switch (element.type) {
-        case 'section':
+        case 'section': {
+          const partialModel = parseFormSubmissionModel(element.elements, model)
+          Object.assign(model, partialModel)
+          break
+        }
```

Comment:
```suggestion
        case 'section':
```

Reply:
that would just change it back to what it was originally?

Reply:
ah, lol. 

If sections are broken, wouldn't the same bug be there for pages as well?

Can you please confirm the same bug does not exist for pages?